backend/ftp/ftp.go

```diff
@@ -158,6 +163,12 @@ func (f *Fs) ftpConnection() (*ftp.ServerConn, error) {
 			InsecureSkipVerify: f.opt.SkipVerifyTLSCert,
 		}
 		ftpConfig = append(ftpConfig, ftp.DialWithTLS(tlsConfig))
+	} else if f.opt.ExplicitTLS {
```

Comment:
Are explicit TLS and TLS mutually incompatible? If so we should flag that to the user by returning an error here.

Reply:
I could imagine that it could work technically. But I could not find any information about it on the web, and I think there is no practical use case for it. An error is now returned when both methods are used together. A note has been added to the documentation.